final_task/rss_reader/converter.py

```diff
@@ -0,0 +1,153 @@
+"""This module converts data to pdf"""
+
+from fpdf import FPDF
+import urllib.request as url
+import os
+import sys
+import datetime
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+font = 'ARIALUNI.TTF'
+
+win_font = r'C:\Windows\Fonts\arial.ttf'
+lin_font = r'/usr/share/fonts/dejavu/DejaVuSansCondensed.ttf'
```

Comment:
same as above

Reply:
All global variables were fixed, it will be updated during the next commit.